app/components/primer/state_component.rb

```diff
@@ -4,12 +4,19 @@ module Primer
   # Component for rendering the status of an item.
   class StateComponent < Primer::Component
     COLOR_DEFAULT = :default
-    COLOR_MAPPINGS = {
+    NEW_COLOR_MAPPINGS = {
+      open: "State--open",
+      closed: "State--closed",
+      merged: "State--merged"
```

Comment:
I think the idea was to keep something like

```suggestion
      open: "State--green",
      closed: "State--red",
      merged: "State--purple"
```

and later on we can change `--green` to `--open`

Reply:
this makes more sense.